ansible/roles/dashd/templates/dash.conf.j2

```diff
@@ -87,7 +87,7 @@ llmqinstantsend=llmq_devnet
 # First 1000 blocks should be mined with lowest difficulty (like in regtest, so it's instant)
 # and first 500 blocks are mined with a block subsidity multiplied by 10
 # this allows to register MNs immediately (DIP3 activates on block 2)
```

Comment:
Will this logic be changed with new difficulty? Shall we change in dashmate as well?

Reply:
I'm not sure, we need input from @PastaPastaPasta or core team about how many blocks are actually necessary for quorum rotation if/when it is merged to their develop branch. I'm not sure if this will be the final activation mechanism for DIP24 or if this is just a temporary fix. I'll remove it from this PR for now.